scripts/assemble.ts

```diff
@@ -74,13 +74,24 @@ ${Object.entries(icon)
 ]);
 `;
 
+    let doc = `
+/**
+ * ${Object.entries(icon)
+   .map(
+     ([weight, path]) =>
+       `@${weight} ![img](data:image/svg+xml;base64,${Buffer.from(`<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 256 256" fill="#000"><rect width="256" height="256" fill="#FFF" rx="40" ry="40"/>${path}</svg>`).toString("base64")})`
```

Comment:
I don't feel great about using `path` here, as the attribute names have been camelCased [here](https://github.com/phosphor-icons/react/blob/master/scripts/index.ts#L64-L81). Maybe `readAssetsFromDisk()` should return a `{ preview: string; jsx: string }` per icon per weight instead of just a `string`. You can do the resizing and adding the backgrounds there. WDYT?

Reply:
It does seem to work in VSCode (and I guess everything Chromium), but its possible this breaks in Firefox in web-based IDEs and stuff. Just would rather not.